us.zoom.Zoom.json

```diff
@@ -44,6 +45,7 @@
                     "type": "script",
                     "dest-filename": "zoom.sh",
                     "commands": [
+                        "ln -s ~/.var/app/us.zoom.Zoom/config/zoomus.conf ~/.var/app/us.zoom.Zoom/.config",
```

Comment:
Does it still work if you use `"ln -s $XDG_CONFIG_HOME ~/.config",` and remove the `"--persist=.config"` again? I think this would be simpler.

Reply:
Let me check

Reply:
It worked! This one is 100000x better